ModuleNotFoundError: No module named 'setuptools' while running github actions

https://github.com/thumbor/thumbor/actions/runs/10958697099/job/30429460166

```
Run docker exec thumbor_test make compile_ext
Traceback (most recent call last):
  File "/app/setup.py", line 1[5](https://github.com/thumbor/thumbor/actions/runs/10958697099/job/30429460166#step:8:6), in <module>
    from setuptools import Extension, setup
ModuleNotFoundError: No module named 'setuptools'
make: *** [Makefile:1[7](https://github.com/thumbor/thumbor/actions/runs/10958697099/job/30429460166#step:8:8): compile_ext] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -14,6 +14,7 @@ setup:
 	@echo  "\n\nYou are strongly recommended to run 'pre-commit install'\n"
 
 compile_ext build:
+	@$(PYTHON) -m pip install setuptools
 	@$(PYTHON) setup.py build_ext -i
 
 test: build redis

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -44,7 +44,6 @@ dependencies = [
   "piexif>=1.1.3,<2.0.0",
   "Pillow>=10.4.0,<13.0.0",
   "pytz>=2023.3.post1,<2027.0.0",
-  "setuptools>=78.1.1,<79.0.0",
   "statsd>=4.0.1,<5.0.0",
   "thumbor-plugins-gifv>=0.1.5,<1.0.0",
   "tornado>=6.4,<7.0.0",

